extensions/wikia/Flags/controllers/templates/SpecialFlagsController_index.php

```diff
@@ -27,9 +27,16 @@
 				</a>
 			</td>
 			<td class="flags-special-list-item-params">
-				<?php foreach ( json_decode( $flag['flag_params_names'] ) as $name => $description ): ?>
-					<?= $name ?> <small><em><?= $description ?></em></small><br>
-				<?php endforeach; ?>
+				<?php
+					$paramsNames = json_decode( $flag['flag_params_names'] );
+					if ( is_array( $paramsNames ) ) :
```

Comment:
`json_decode` without setting true as the second parameter returns an StdClass object, not an array, so this check will always fail and no parameters will be displayed. Fixed in https://github.com/Wikia/app/pull/7652.